plasmapy/particles/particle_class.py

```diff
@@ -1005,6 +1005,24 @@ def isotope(self) -> str | None:
         """
         return self._attributes["isotope"]
 
+    @property
+    def nucleus(self) -> Particle:
+        """
+        Return the nucleus of an atom.
+
+        Returns
+        -------
+        `~plasmapy.particles.exceptions.InvalidElementError`
+            If the particle is not an element, isotope, or ion.
+        """
+        if not self.element:
+            errmsg = (
+                f"Unable to return the nucleus of {self.symbol} because "
+                "it is not an element or isotope."
+            )
+            raise InvalidElementError(errmsg)
+        return Particle(self.isotope or self.element, Z=self.atomic_number)
```

Comment:
This is short for `self.isotope if self.isotope else self.element`.  It was a refactoring suggested by Sourcery.